man/man8/zfs.8

```diff
@@ -461,7 +461,16 @@ If needed, ZFS file systems can also be managed with traditional tools
 If a file system's mount point is set to
 .Sy legacy ,
 ZFS makes no attempt to manage the file system, and the administrator is
-responsible for mounting and unmounting the file system.
+responsible for mounting and unmounting the file system. Because pools must
+be imported before a legacy mount can succeed, administrators should ensure
+that legacy mounts are only attempted after the zpool import process
+finishes at boot time. For example, on machines using systemd, the mount
+option
+.Pp
+.Nm x-systemd.after=zfs-import.target
```

Comment:
is only available since systemd 233, which excludes current Debian stable and Ubuntu LTS (I have not idea about RH and derived distros).

maybe use the (better fitting as well) `x.systemd.requires`, which adds a `Requires` and `After`? a reference to `man systemd.mount` might also be helpful for further reading.

Reply:
I'll do this.